assets/html/scss/documenter/layout/_main.scss

```diff
@@ -125,6 +126,16 @@
       .docs-footer-nextpage {
         text-align: right;
       }
+
+      .flexbox-break {
+        flex-basis: 100%;
+        height: 0;
+      }
+
+      .footer-message {
+        font-size: 0.9em;
```

Comment:
I would make the font just a tiny bit smaller, so it would be slightly less intrusive visually:

```suggestion
        font-size: 0.7em;
```

![footer](https://user-images.githubusercontent.com/147757/87486053-6b88bb00-c68e-11ea-97ed-c48f4a71647c.png)